src/pages/ProductPage/ProdDetail/ProdDetail.scss

```
@@ -0,0 +1,28 @@
+.prodDetail{
+  font-weight: bold;
+
+  h1{
+    font-size: 24px;
+  }
+  
+  h2{
+    color: orange;
+    font-size: 12px;
+  }
+
+  .description{
+    font-weight: 200;
+  }
+
+  .price{
+    span{

```

If you doesn't have any rule inside `.price` but has in `span` you can put both at the same line:

```suggestion
  &__price span {
```